hs_core/templates/debug/debug_resource.html

```diff
@@ -70,31 +70,35 @@ <h2>Resource Flags</h2>
   </tbody>
 </table>
 
+<div id="non-preferred-paths">
```

Comment:
Testing the debug page never resolved for me locally.
I haven't used the debug page before, but I think this isn't a regression...I think it never worked locally?
![image](https://user-images.githubusercontent.com/17934193/212364532-d3fbc1d2-4155-4619-a82c-96e55e29731b.png)